fms2_io/netcdf_io.F90

```diff
@@ -149,6 +149,11 @@ module netcdf_io_mod
   character (len=20) :: time_name
   type(dimension_information) :: bc_dimensions !<information about the current dimensions for regional
                                                !! restart variables
+  logical :: use_collective = .false. !< Flag indicating if we should open the file for collective input
+                                      !! this should be set to .true. in the user application if they want
+                                      !! collective reads (put before open_file())
```

Comment:
@uramirez8707 @thomas-robinson - does this comment/variable name make it clear this applies only to reads and not writes?

Reply:
It says *for collective input* so I think that's clear